third_party/mustache/README.amp

```diff
@@ -0,0 +1,10 @@
+URL: https://github.com/janl/mustache.js/
+License: The MIT License
+License File: https://github.com/janl/mustache.js/blob/master/LICENSE
+
+Description:
+Fork of mustache.js library.
+
+Local Modifications:
+None yet, but expected changes for restricting instructions and sanitizing
```

Comment:
Will definitely do this via a separate PR + changes. And I do hope that we can just reference it via node_modules/, but I will first have to send a pull request to that project, so I hope I can first get a solid prototype going with a fork to actually figure out what might be missing in the original library. Upstream security possibility is also strong enough to warrant the fork.